src/pe/exception.rs

```diff
@@ -624,6 +684,20 @@ impl<'a> UnwindInfo<'a> {
             },
         }
     }
+
+    /// Returns an iterator over C scope table entries in this unwind info.
+    ///
+    /// If this unwind info has no [UnwindHandler::ExceptionHandler], this will always return `None`.
+    pub fn c_scope_table_entries(&self) -> Option<ScopeTableIterator<'a>> {
+        let UnwindHandler::ExceptionHandler(_, data) = self.handler? else {
+            return None;
+        };
```

Comment:
nice use of let else + ? in Option returning func :) it did throw me off at first, i hadn't seen this pattern together at first so I parsed it as a double Option, then I realized handler is an Optional enum.